### PR TITLE
Initial setup of ST pipeline

### PR DESCRIPTION
**Pull request by rylandos**
## Checklist:

- [ ] It contains only changes required by issue (does not contain other PR)
- [ ] Includes a link to an issue (if apply)
- [ ] I have added tests to cover my changes.

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -346,6 +345,267 @@ steps:
...
+      GIT_PRIVATE_SSH:
+        from_secret: github_private_key
+      TF_VAR_s3_aws_access_key_id:
+        from_secret: aws_access_key_id
+      TF_VAR_s3_aws_secret_access_key:
+        from_secret: aws_secret_access_key
+      TF_VAR_build_number: ${DRONE_BUILD_NUMBER}
+      TF_VAR_secret_key_base:
+        from_secret: mod_ruby_secret
+      TF_VAR_google_analytics_id: UA-144774549-1
```
Comment (gregb-informed) [inline .drone.yml]: Check that this and the feedback url is correct

Comment (rylandos) [reply .drone.yml]: @gregb-informed do you know where these details can be found?

Comment (gregb-informed) [reply .drone.yml]: probably need to ask @kingarj

Comment (kingarj) [reply .drone.yml]: Looking at the SIT steps, GA ID should be UA-144774549-3 and feedback URL should be https://www.surveymonkey.co.uk/r/HJ9FKJC.

**Pull request merged**